### PR TITLE
Add OpenSign™

### PR DESCRIPTION
**Pull request by nxglabs**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/opensign.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+labels: addition
+assignees: ''
+
+---
+
+Thanks for taking the time to suggest an addition to awesome-selfhosted! 
+
+Please fill out information below (all fields are mandatory unless noted otherwise):
+
+```yaml
```
Comment (Ki-er) [inline software/opensign.yml:14]:
```suggestion
```

File: software/opensign.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):
+- [x] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
+- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
+- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
+- [x] Any software project you are adding to the list is actively maintained.
+- [x] Any software project you are adding was first released more than 4 months ago.
+- [x] Any software project you are adding has working installation instructions.
+- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
```
Comment (Ki-er) [inline software/opensign.yml:47]:
```suggestion
```
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Please also remove the comments.

**Comment by nxglabs** [comment]
@Ki-er now that the project is more than 4 months old, please add it.

File: software/opensign.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: "OpenSign™"
```
Comment (Ki-er) [inline software/opensign.yml:1]:
```suggestion
name: "OpenSign"
```

File: software/opensign.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted DocuSign alternative."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - MongoDB
+  - Docker
```
Comment (Ki-er) [inline software/opensign.yml:10]: See below: Mongodb isnt  platform but does require 3rd party flag.
```suggestion
platforms:
  - Nodejs
  - Docker
```

File: software/opensign.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+  - Document Management
+  - Digital Signature
+  - e-signature
+  - DocuSign alternative
+  - Free digital signatures
```
Comment (Ki-er) [inline software/opensign.yml:16]: Tags are predefined and can be found here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags
Depends on 3rd party due to MongoDB

```suggestion
tags:
  - Document Management
depends_3rdparty: true
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Some changes to be made.

**Review by Ki-er — APPROVED** [review]
Looks good to me!

File: software/opensign.yml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+name: "OpenSign"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted DocuSign alternative."
```
Comment (nodiscc) [inline software/opensign.yml:4]: The description should explain what the software does (you can add `(alternative to DocuSign)` at the end of the description)

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/opensign.yml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+name: "OpenSign"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted document signing software. Alternative to DocuSign."
```
Comment (Ki-er) [inline software/opensign.yml:4]: Syntax for alternatives can be found here:https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

```suggestion
description: "Free, open source & self-hosted document signing software (alternative to DocuSign)."
```

**Review by nodiscc — APPROVED** [review]

**Pull request merged**